packages/nc-gui/pages/projects/index/index.vue

```diff
@@ -25,7 +24,7 @@ const openProject = async (project: ProjectType) => {
   $e('a:project:open', { count: projects.length })
 }
 
-const formatTitle = (title: string) =>
+const formatTitle = (title: '') =>
```

Comment:
type should be  `string`?

Reply:
`''` will implicitly cast as `string`. I think the idea here is in case `title` is null for some reason, it won't throw error.

Reply:
<img width="723" alt="Screenshot 2022-12-03 at 11 15 31 AM" src="https://user-images.githubusercontent.com/61551451/205426293-e430656a-0225-4907-ac06-bf72b99a24c9.png">